docs/en/basic/Command-Line-Interface.md

```diff
@@ -695,6 +695,23 @@ For example, `setfacl` can be used to give read and execute permissions to a use
 $ ./bin/alluxio fs setfacl -m "user:testuser:r-x" /testdir/testfile
 ```
 
+### setReplication
+
+The `setReplication` command sets the max and/or min replication level of a file or all files under
+a directory recursively. This is a metadata operation and will not cause any replication to be
+created or removed immediately. The replication level of the target file or directory will be
+changed automatically in background. This command takes an argument of `--min` to specify the
+minimal replication level and -max for the maximal replication. Specify -1 as the argument of
```

Comment:
should this be `--max` instead of `-max`?

Reply:
good  catch!